src/main/java/wellnus/atomichabit/feature/AtomicHabitUi.java

```diff
@@ -0,0 +1,40 @@
+package wellnus.atomichabit.feature;
+
+import java.io.InputStream;
+
+import wellnus.ui.TextUi;
+
+/**
+ * This section is to be updated with main UI class
```

Comment:
Add '.' at the end of sentence

Reply:
Also a bit unhelpful in my opinion. Maybe write something like 'Provides a customised interface and output message formatting for the atomic habit feature.'